MIP63/MIP63.md

```diff
@@ -77,7 +77,7 @@ The only connection is through `dss-vest`, where Keeper Networks are allocated a
 
 #### Keeper Duties and Expectations
 
-It is expected that Keeper Networks will be able to appropriate stream in their budget to be within the requested buffer min and max values. Any excess DAI beyond the max should be returned to the surplus buffer, and the network should act in a timely manor to replenish DAI if the buffer drops below the min value. If the DAI in the buffer consistently is dropping below the min value then a new application should be submitted to increase the budget.
+It is expected that Keeper Networks will be able to appropriate stream in their budget to be within the requested buffer min and max values. Any excess DAI beyond the max should be returned to the surplus buffer, and the network should act in a timely manner to replenish DAI if the buffer drops below the min value. If the DAI in the buffer is consistently running out then a new application should be submitted to increase the budget.
```

Comment:
I think there should be a maximum duration for these budget requests. I'd suggest 12 months but open to other suggestions.

```suggestion
It is expected that Keeper Networks will be able to appropriate stream in their budget to be within the requested buffer min and max values. Any excess DAI beyond the max should be returned to the surplus buffer, and the network should act in a timely manner to replenish DAI if the buffer drops below the min value. If the DAI in the buffer is consistently running out then a new application should be submitted to increase the budget. Budget requests should not exceed a maximum duration of 12 months.
```

Reply:
6 months is pretty small. Ideally this should eventually be set it and forget it.

Reply:
Initially I'd written 12 months, happy for you to pick an alternative number. The issue with set it and forget it is that it minimizes opportunity for Governance oversight. The act of requesting a budget forces Governance to assess whether the keeper is providing value or not.

Reply:
done